manual/3-functions.md

```diff
@@ -48,4 +48,14 @@ The bottom LED close to the USB port indicates if the load output and USB port a
 
 The three middle LEDs indicate the state of charge (SOC) of the connected battery. More LEDs means more energy is present in the battery.
 
+| LED ID | ON                            | 
+|--------|-------------------------------| 
+| LED 1  | battery SOC low               | 
+| LED 2  | battery SOC middle            | 
+| LED 3  | battery SOC high              | 
+| LED 4  | Load and USB port enabled     | 
+| LED 5  | free for costum configuration | 
```

Comment:
Did you double-check whether this renders correctly if built with Pandoc? They have some special way to configure tables IIRC. And unfortunately it's not standardized in Markdown.

Just wondering because the table in the specs page is formatted differently.

Reply:
I'm not sure if the new suggestion works. Did a quick check and for some reason pandoc doesn't want to create the PDF at all. For the HTML page even the first suggestion seems to work.

However, the content looks wrong. If LED1 is on, would the SOC not be high?